Assets/Scripts/Gamestate/PlayerManager.cs

```diff
@@ -99,10 +98,10 @@ public void SetPlayerInput(FPSInputManager playerInput)
         canvas.planeDistance = 0.11f;
         
         // Set player color
-        // var meshRenderer = meshBase.GetComponentInChildren<SkinnedMeshRenderer>();
-        // var ragdollRenderer = ragdoll.GetComponentInChildren<SkinnedMeshRenderer>();
-        // meshRenderer.materials[0].SetColor("_Color", identity.color);
-        // ragdollRenderer.materials[0].SetColor("_Color", identity.color);
+         //var meshRenderer = meshBase.GetComponentInChildren<SkinnedMeshRenderer>();
+         //var ragdollRenderer = ragdoll.GetComponentInChildren<SkinnedMeshRenderer>();
+         //meshRenderer.materials[0].SetColor("_Color", identity.color);
+         //ragdollRenderer.materials[0].SetColor("_Color", identity.color);
```

Comment:
Uhhh, so we still don't have coloured players back?

Reply:
> - Color changing still WIP.
@ 832a650be5c32cc2265e81b212c0bd18ffcd7a28

We basicly need to use the custom material with a color changing field, something I haven't had time to do yet